pkg/utils/kubernetes/controllers.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetes
+
+import versionutils "github.com/gardener/gardener/pkg/utils/version"
+
+// APIGroupControllerMap is a map for the Kubernetes API groups and the corresponding controllers for them.
+var APIGroupControllerMap = map[string]map[string]versionutils.VersionRange{
```

Comment:
This map has several mismatches to https://github.com/gardener/gardener/blob/cd970e50baee6ac66556a655ed0df467bd94ae8b/pkg/utils/validation/apigroups/apigroups.go#L30.

For example, if disabling `resource/v1alpha2` you have to specify the following:
```
...
kubeApiServer:
  runtimeConfig:
    runtimeConfig:
      resource.k8s.io/v1alpha2: false
```

This doesn't disable the `resource-claim-controller` controller.

Reply:
Ah, yes. Thanks for noticing. Please check https://github.com/gardener/gardener/pull/8763/commits/a0dc7e774da37491e9b818458b89c90194bb87fb